templates/pod/README$.md.hbs

```diff
@@ -0,0 +1,22 @@
+# Welcome to your pod
```

Comment:
Do we want this to be an index.html so it's the first thing they see when they go to their pod URI?

Reply:
No; I had tried that but this breaks the automated directory listing. Instead, that listing no shows a README.